satpy/writers/__init__.py

```diff
@@ -271,6 +272,7 @@ def to_image(dataset, copy=True, **kwargs):
 
 
 class Writer(Plugin):
+
```

Comment:
Is this really PEP8? I've seen @mraspaud do this, but I thought it was against typical documentation standards. I don't think most packages do this. Same with the blank line between class declaration and the init method on 441.

Reply:
https://www.python.org/dev/peps/pep-0008/#blank-lines

It says:

"Method definitions inside a class are surrounded by a single blank line."

I have no strong opinion. I find my auto-pep8 useful, and if it is not making the code look worse I am satisfied!

Reply:
Which means it should have been:
```
class Writer(Plugin):

    """Writer plugins. They must implement the *save_image* method. This is an
    abstract class to be inherited.
    """

    def __init__(self,
                 name=None,
                 fill_value=None,
                 file_pattern=None,
                 base_dir=None,
                 **kwargs):

         # Load the config
```

It doesn't actually say anything about having doc strings immediately after a class definition.

Reply:
I'm 90% sure the first line of the docstring should only be a single line and shouldn't wrap to the next line. Also we are following google style docstrings (or trying to): http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Not that their examples are law, but still. It just seems like wasted space.

Reply:
Agree with @davidh-ssec here, the first line shouldn't wrap.